crates/nu-cli/src/cli.rs

```diff
@@ -79,6 +79,7 @@ pub fn create_default_context(
 
         context.add_commands(vec![
             // System/file operations
+            whole_stream_command(Du),
```

Comment:
Am I reading this right, `whole_stream_command(Du)` is in here twice?

Reply:
good find, let me fix it